docs/source/debugging-performance.rst

```diff
@@ -0,0 +1,15 @@
+Debugging and Performance
+==========================
+
+This section contains resources to help you debug and understand performance.
+
+
+.. toctree::
+   :maxdepth: 1
+
+   how-to/debug.rst
+   Visualize Task graphs <graphviz.rst>
+   Diagnostics Local <diagnostics-local.rst>
+   Dashboard <dashboard.rst> 
+   Diagnostics Distributed <diagnostics-distributed.rst>
+   Phases of computation <phases-of-computation.rst>
```

Comment:
I think we can make these sections more distinct by putting dashboard earlier and putting the diagnostics pages after it. I know they aren't quite that separate yet, but that makes sense to me as a direction we can go.

```suggestion
   how-to/debug.rst
   Visualize task graphs <graphviz.rst>
   Dashboard <dashboard.rst> 
   diagnostics-local.rst
   diagnostics-distributed.rst
   Phases of computation <phases-of-computation.rst>
```